### PR TITLE
General WebExt improvements

### PR DESCRIPTION
**Pull request by DaAwesomeP**
- ~~Set minimum version to current FF pre-v57 Nightly~~ (see [discussion comment](https://github.com/arantius/resurrect-pages/pull/28/files/5ab870b7a55a0635558e2491f448d7e5f94526b9#diff-4b1eb3dc48c4e16d49db5b42298fe654))
 - Fixed bug where background tab option in context menu was never selected
 - Compare with triple equals (yes, it is only ever set to a number, but a good practice)
 - Add `shortname` to manifest (needed for Chrome/Opera extensions if you plan to do that too)
 - Added homepage URL to manifest
I would recommend adding the `developer` and `author` keys to the manifest as well.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
   "description": "__MSG_extensionDescription__",
   "version": "4",
   "default_locale": "en",
+  "homepage_url": "https://github.com/arantius/resurrect-pages",
 
   "applications": {
     "gecko": {
       "id": "{0c8fbd76-bdeb-4c52-9b24-d587ce7b9dc3}",
-      "strict_min_version": "57.0"
+      "strict_min_version": "57.0a1"
```
Comment (arantius) [inline manifest.json]: AIUI addons.mozilla.org will reject this.

Comment (DaAwesomeP) [inline manifest.json]: @arantius OK, I'll revert that!

Comment (DaAwesomeP) [inline manifest.json]: @arantius reverted

**Comment by arantius** [comment]
I've cherry picked the bits of this I like.

**Pull request closed without merging**